reference/6/Microsoft.PowerShell.Utility/ConvertFrom-Json.md

```diff
@@ -77,6 +78,14 @@ Then it uses the pipeline operator to send the delimited string to the **Convert
 
 The Join operator is required, because the **ConvertFrom-Json** cmdlet expects a single string.
 
+### Example 4: Convert a JSON string to a hash table
```

Comment:
I think it would be more clear to a novice user to separate these into two examples

Reply:
I'll remove the 2nd one (key that is empty string) because this is also documented in the switch documentation and is not so useful as an example